app/workers/retirement_worker.rb

```diff
@@ -11,6 +11,7 @@ def perform(count_id)
       workflow = count.workflow
       WorkflowRetiredCountWorker.perform_async(workflow.id)
       PublishRetirementEventWorker.perform_async(workflow.id)
+      UnfinishWorkflowWorker.perform_async(workflow.id)
```

Comment:
If this needs to be checked here, it'll need to be called from `WorkflowRetiredCountWorker` (like #1950) to get the current counts.